modules/doi/modules/doi_importer/includes/importer.inc

```diff
@@ -24,38 +24,44 @@ class DOIImporter extends IslandoraBatchImporter {
    *
    * @see IslandoraBatchImporter::getForm()
    */
-  public static function getForm(array &$form_state) {
-    return array(
-      'fs' => array(
-        '#type' => 'fieldset',
-        '#title' => t('DOI Batch Journal Importer'),
-        '#description' => t('Provide either a list of Digital Object Identifiers in a file, or enter DOIs in the textarea below. In either case, the DOIs in the list should be separated by either whitespace or commas (or some combination thereof). NOTE: Presently, only journal (articles) are supported.'),
-        'file' => array(
-          '#type' => 'managed_file',
-          '#title' => t('File of DOIs'),
-          '#description' => t('A file listing Digital Object Identifiers (with ".txt" extension).'),
-          '#upload_validators' => array(
-            'file_validate_extensions' => array('txt'),
+  public static function getForm(array & $form_state) {
+    $openurl_pid = variable_get('islandora_doi_openurl_pid', 'user@example.com');
```

Comment:
@bryjbrown I get it. We need more documentation for this probably. Will be good to get that sorted after code freeze. I wonder if we could do some test based on 
https://github.com/Islandora/islandora_scholar/blob/7.x/modules/doi/includes/utilities.inc#L38-L50
Basically, something that allows us to be sure that the stored email will work, and if so, let the whole thing pass. Just because, without knowing I could be tempted to put any email (not registered) to test wich would end triggering again a silent response here...

For now i will say ok, but we should add a note to the jira ticket  and open a new one that solves this for good. DOI-s are quite popular!